tests/test_processor.py

```diff
@@ -117,7 +117,7 @@ def postprocess(self, accumulator):
         proc.process,
         None,
     )
-    assert out == {"itemsum": 45}
+    assert out == ({"itemsum": 45}, 0)
```

Comment:
Based on the fact that `__call__` preserves the old behavior, why did this end up changing?

Reply:
Runner.__call__ preservers old behaviour, this is invoking the exec directly